### PR TITLE
Prettier

### PR DESCRIPTION
**Pull request by allanchau**
This PR updates prettier and runs it across all bin files.

### Testing

- [ ] Link this PR and try a few commands, nothing has changed so it should work as it did before.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,7 +1,4 @@
 {
-    "extends": ["@idearium/eslint-config", "plugin:prettier/recommended"],
-    "parserOptions": {
-        "ecmaVersion": 2017
-    },
-    "root": true
+    "root": true,
```
Comment (smebberson) [inline .eslintrc.json:2]: @allanchau, don't we need `prettier/recommended` to remove styling rules from our config?

Comment (allanchau) [reply .eslintrc.json:2]: @smebberson I've fixed that up now.

Comment (smebberson) [reply .eslintrc.json:2]: @allanchau, also, why did you remove the `ecmaVersion`? I'm sure that was only added if required? I think it's to allow `async () => {}`. You can check it being incorrectly reported in `bin/lib/c.js` at L339.

Comment (allanchau) [reply .eslintrc.json]: @smebberson I don't remember why I changed this file, I probably copied it from another project. I've reverted the change now.

Comment (smebberson) [reply .eslintrc.json]: Cool.

**Pull request merged**